Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -732,8 +774,36 @@ struct SpecialCasing {
     Condition condition { Condition::None };
 };
 
+enum class CompatibilityFormattingTag : u8 {
+    Canonical,
+    Font,
+    NoBreak,
+    Initial,
+    Medial,
+    Final,
+    Isolated,
+    Circle,
+    Super,
+    Sub,
+    Vertical,
+    Wide,
+    Narrow,
+    Small,
+    Square,
+    Fraction,
+    Compat
+};
+
+struct CodePointDecomposition {
+    u32 code_point { 0 };
+    CompatibilityFormattingTag tag { CompatibilityFormattingTag::Canonical };
+    Span<u32 const> decomposition;
+};
+
 Optional<Locale> locale_from_string(StringView locale);
 
+Span<CodePointDecomposition const> code_point_decompositions();
```

Comment:
Which, since the next commit adds Normalize.cpp anyways, it'd be a bit better if both `code_point_decomposition` and `code_point_decompositions` were declared in Normalize.h (with weak definitions in Normalize.cpp) to be closer to where it's actually used.

Reply:
Done in 784beff :+1: